bin/update-contributors.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+cat > CONTRIBUTORS.md <<EOS
+# Cumulus Contributors
+
+EOS
+
+(
+  echo 'Alireza Jazayeri'
```

Comment:
Odd, is he not in the commit list because <git configuration> didn't have a name?

Reply:
His git config.name was set to "scisco".